File: FootballApp/Models/Team.cs
namespace FootballApp.Models
{
    public class Team
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Adress { get; set; }
        public string Email { get; set; }
        public string Telephone { get; set; }
        public int MangerId { get; set; }
        public Manager Manager { get; set; }
        public string PhotoUrl { get; set; }
        public int LeagueId { get; set; }
        public League League { get; set; }
    }
}

File: FootballApp-SPA/src/app/_models/Team.ts
import { Manager } from './Manager';
import { League } from './league';

export interface Team {
    id: number;
    name: string;
    address: string;
    email: string;
    telephone: string;
    managerId: number;
    manager: Manager;
    photoUrl: string;
    leagueId: number;
    league: League;
}
